README.md

```diff
@@ -17,11 +17,13 @@ Please find the sample that fits your use-case from the table below.
 
 | Sample | Description | Use-Case |
 |--------|-------------|----------|
-| [Okta-Hosted Login](/samples-aspnetcore-3x/okta-hosted-login) | An application server that uses the hosted login page on your Okta org, then creates a cookie session for the user in the ASP.NET Core application. | Traditional web applications with server-side rendered pages. |
+| [Federated Auth](/samples-aspnetcore-3x/federated-auth) | An application server that uses the hosted login page on your Okta org, then creates a cookie session for the user in the ASP.NET Core application. | Traditional web applications with server-side rendered pages. |
 | [Resource Server](/samples-aspnetcore-3x/resource-server) | This is a sample API resource server that shows you how to authenticate requests with access tokens that have been issued by Okta. | Single-Page applications. |
 | [Self-Hosted Login](/samples-aspnetcore-3x/self-hosted-login) |  An application server that uses a self-hosted login page on your ASP.NET Core application. | Traditional web applications with server-side rendered pages. |
 | [Social Login](/samples-aspnetcore-3x/social-login) |  An application server that uses a self-hosted login page with multiple login options on your ASP.NET Core application. | Traditional web applications with server-side rendered pages. |
 
+:information_source: Federated Okta Hosted Login is used when you want Okta to handle authentication through its login page, hosted in Okta’s ecosystem.
```

Comment:
Nit: Should we talk about SSO between apps here since Okta cookie can be used for this purpose? Applications can share session using Okta session cookie which wouldn't be possible in self hosted login.